lib/membrane_mp3_mad_plugin/decoder.ex

```diff
@@ -3,17 +3,18 @@ defmodule Membrane.MP3.MAD.Decoder do
   Decodes MPEG audio to raw data in S24LE format
   """
   use Membrane.Filter
+  require Membrane.Logger
+
   alias Membrane.Caps.Audio.{Raw, MPEG}
   alias __MODULE__.Native
-  alias Membrane.Buffer
-  use Membrane.Log
+  alias Membrane.{Buffer, Logger}
 
-  def_input_pad :input, demand_unit: :buffers, caps: [:any, MPEG]
+  def_input_pad :input, demand_mode: :auto, caps: [:any, MPEG]
```

Comment:
I think it should accept RemoteStream instead of any

Reply:
Well, this means the file element should send it - right now it doesn't and defines output caps as `:any`. I'll create an issue for that